vmc/utils.go

```diff
@@ -4,6 +4,7 @@
 package vmc
 
 import (
+	"github.com/google/uuid"
```

Comment:
I haven't checked differences between "google/uuid" and "satori/go.uuid", but the satori one is being used here - https://github.com/terraform-providers/terraform-provider-vmc/blob/master/vmc/resource_vmc_public_ip.go#L13
We should try to use one uuid lib across the provider for consistency.

Reply:
@shylasrinivas I agree we should use the same library. I missed the one we were using in vmc_public_ip.go. We can probably change the usage in vmc_public_ip.go from    There is a comment here https://stackoverflow.com/questions/25051675/how-to-validate-uuid-v4-in-go where someone has compared satori/go.uuid vs google/uuid and has found that Google's UUID check is 3.3x faster than this one. I will update the library we are using in vmc_public_ip.go

Reply:
For our use case (since uuid lib is used minimally), these speeds don't matter much. I am ok with google/uuid, as long as we use any one, instead of both.

Reply:
Done